update api and change server setup docker command (#4)
PDFTron/webviewer-BIM-sample: README.md

webviewerBIM.Viewer.enableSSAO();
```

### disableSSAO()

Call `disableSSAO` to disable screen-space ambient occlusion for the viewer.

```js
const webviewerBIM = await initializeBimViewer(instance, serverURL, options);
webviewerBIM.Viewer.disableSSAO();
```

### setSSAOOptions()

Call `setSSAOOptions` to adjust screen-space ambient occlusion for the viewer.

```js
const webviewerBIM = await initializeBimViewer(instance, serverURL, options);
webviewerBIM.Viewer.setSSAOOptions({
 // example parameters：
 isDynamicRadius: true,
 radius: 1,
 loops: 64,
 blurRadius: 2,
 power: 1.4,
})
```

### enableAntiAliasing()

Call `enableAntiAliasing` to enable anti-aliasing for the viewer.

```js
const webviewerBIM = await initializeBimViewer(instance, serverURL, options);
webviewerBIM.Viewer.enableAntiAliasing();
```

### disableAntiAliasing()

Call `disableAntiAliasing` to disable anti-aliasing for the viewer.

```js
const webviewerBIM = await initializeBimViewer(instance, serverURL, options);
webviewerBIM.Viewer.disableAntiAliasing();
```

### setCameraSensitivity(number)

Call `setCameraSensitivity` to set the sensitivity for Orbit/Pan tool.
- `number` to set the sensitivity

```js

> const tool = instance.Core.documentViewer.getToolMode();

Reviewer: Hmm now that I think about this - it assumes the current selected tool is Orbit or Pan.
@JingyeHou @NoftScarlet Do we have any documentation for the BIM tool names?
```
const toolMap = {
  orbit: 'Orbit3D',
  pan: 'Pan3D',
  walk: 'Walk',
};
```

I think it's better if we have the documentation say:
```
const tool = instance.Core.DocumentViewer.getTool(toolMap.orbit);
```

tool.setCameraSensitivity(10);
```

### getCameraSensitivity()

Call `getCameraSensitivity` to get the sensitivity for Orbit/Pan tool.
Returns a value of Number

```js

> const tool = instance.Core.documentViewer.getToolMode();

Reviewer: See comment above about tool names.

webviewerBIM.File.load3dAsset('Add URL to your 3D asset here');
```

### unmountBimViewer()

> Call `unmountBimViewer` to revert the WebViewer back and clear memory.

Reviewer: Maybe a bit more detail here.

```suggestion
Call `unmountBimViewer` to revert WebViewer back to its original state, and to clear any memory from the WebViewer BIM client.
```

```js
import  Webviewer  from  '@pdftron/webviewer';

> import { initializeBimViewer, unmountBimViewer } from '@pdftron/webviewer/bim-client'

Reviewer: ```suggestion
import { initializeBimViewer, unmountBimViewer } from '@pdftron/webviewer-bim-client'
```

Webviewer({
  path: '/webviewer/lib',
}, document.getElementById('viewer')).then(instance  => {
  const  license = `---- Insert commercial license key here after purchase ----`;
  const  serverURL = `---- Insert server URL after setup ----`;
  const  options = {
    license: license,
  }

  const WebViewerBIM = await initializeBimViewer(instance, serverURL, options);

>   webviewerBIM.File.load3dAsset(

Reviewer: I think this can stay on one line.
```suggestion
  webviewerBIM.File.load3dAsset("Add URL to your 3D asset here");
```

    "Add URL to your 3D asset here"
  );

>   // input your code here

Reviewer: Let's add a newline for spacing and change the comment to explain they can call `unmountBimViewer` when they're ready to unmount. Also comment out `unmountBimViewer(instance);` otherwise someone running the sample will accidentally unmount immediately after initializing. I see it causing confusion.

```suggestion
 
 // Call unmountBimViewer when you're ready to unmount.
 // unmountBimViewer(instance);
```

  unmountBimViewer(instance)
}
```

### enableSSAO()

> Call `enableSSAO` to enable Sub Surface Occulusion for the Viewer.

Reviewer: SSAO stands for `Screen Space Ambient Occlusion`. https://en.wikipedia.org/wiki/Screen_space_ambient_occlusion

```suggestion
Call `enableSSAO` to enable screen-space ambient occlusion for the viewer.
```

```js
const webviewerBIM = await initializeBimViewer(instance, serverURL, options);
webviewerBIM.Viewer.enableSSAO();
```

### disableSSAO()

> Call `disableSSAO` to disable Sub Surface Occulusion for the Viewer.

Reviewer: Same as above. SSAO stands for Screen Space Ambient Occlusion. https://en.wikipedia.org/wiki/Screen_space_ambient_occlusion

```suggestion
Call `disableSSAO` to disable screen-space ambient occlusion for the viewer.
```

```js
const webviewerBIM = await initializeBimViewer(instance, serverURL, options);
webviewerBIM.Viewer.disableSSAO();
```

### setSSAOOptions()

> Call `setSSAOOptions` to adjust Sub Surface Occulusion for the Viewer.

Reviewer: Same as above. SSAO stands for Screen Space Ambient Occlusion. https://en.wikipedia.org/wiki/Screen_space_ambient_occlusion

```suggestion
Call `setSSAOOptions` to adjust screen-space ambient occlusion for the viewer.
```

```js
const webviewerBIM = await initializeBimViewer(instance, serverURL, options);
webviewerBIM.Viewer.setSSAOOptions({
 // example parameters：
 isDynamicRadius: true,
 radius: 1,
 loops: 64,
 blurRadius: 2,
 power: 1.4,
})
```

### enableAntiAliasing()

> Call `enableAntiAliasing` to enable Anti Aliasing for the Viewer.

Reviewer: No need to capitalize these words.
```suggestion
Call `enableAntiAliasing` to enable anti-aliasing for the viewer.
```

```js
const webviewerBIM = await initializeBimViewer(instance, serverURL, options);
webviewerBIM.Viewer.enableAntiAliasing();
```

### disableAntiAliasing()

> Call `disableAntiAliasing` to disable Anti Aliasing for the Viewer.

Reviewer: No need to capitalize these words.
```suggestion
Call `disableAntiAliasing` to disable anti-aliasing for the viewer.
```

webviewerBIM.File.load3dAsset('Add URL to your 3D asset here');
```

### unmountBimViewer()

Call `unmountBimViewer` to revert WebViewer back to its original state, and to clear any memory from the WebViewer BIM client.

```js
import  Webviewer  from  '@pdftron/webviewer';
import { initializeBimViewer, unmountBimViewer } from '@pdftron/webviewer-bim-client'

Webviewer({
  path: '/webviewer/lib',
}, document.getElementById('viewer')).then(instance  => {
  const  license = `---- Insert commercial license key here after purchase ----`;
  const  serverURL = `---- Insert server URL after setup ----`;
  const  options = {
    license: license,
  }

  const webviewerBIM = await initializeBimViewer(instance, serverURL, options);
  webviewerBIM.File.load3dAsset("Add URL to your 3D asset here");

>     "Add URL to your 3D asset here"

Reviewer: Remove this lines 166 and 167.

  );
 
 // Call unmountBimViewer when you're ready to unmount.
 // unmountBimViewer(instance);

>   unmountBimViewer(instance)

Reviewer: Remove this line.

webviewerBIM.Viewer.enableSSAO();
```

### disableSSAO()

Call `disableSSAO` to disable screen-space ambient occlusion for the viewer.

```js
const webviewerBIM = await initializeBimViewer(instance, serverURL, options);
webviewerBIM.Viewer.disableSSAO();
```

### setSSAOOptions()

Call `setSSAOOptions` to adjust screen-space ambient occlusion for the viewer.

```js
const webviewerBIM = await initializeBimViewer(instance, serverURL, options);
webviewerBIM.Viewer.setSSAOOptions({
 // example parameters：
 isDynamicRadius: true,
 radius: 1,
 loops: 64,
 blurRadius: 2,
 power: 1.4,
})
```

### enableAntiAliasing()

Call `enableAntiAliasing` to enable anti-aliasing for the viewer.

```js
const webviewerBIM = await initializeBimViewer(instance, serverURL, options);
webviewerBIM.Viewer.enableAntiAliasing();
```

### disableAntiAliasing()

Call `disableAntiAliasing` to disable anti-aliasing for the viewer.

```js
const webviewerBIM = await initializeBimViewer(instance, serverURL, options);
webviewerBIM.Viewer.disableAntiAliasing();
```

### setCameraSensitivity(number)

Call `setCameraSensitivity` to set the sensitivity for Orbit/Pan tool.
- `number` to set the sensitivity

```js

> const tool = instance.docViewer.getToolMode()

Reviewer: ```suggestion
const tool = instance.Core.documentViewer.getToolMode();
```

> tool.setCameraSensitivity(10)

Reviewer: ```suggestion
tool.setCameraSensitivity(10);
```

```

### getCameraSensitivity()

Call `getCameraSensitivity` to get the sensitivity for Orbit/Pan tool.
Returns a value of Number

```js

> const tool = instance.docViewer.getToolMode()

Reviewer: ```suggestion
const tool = instance.Core.documentViewer.getToolMode();
```

> tool.getCameraSensitivity()

Reviewer: ```suggestion
tool.getCameraSensitivity();
```
